Cargo.toml

```diff
@@ -31,28 +31,39 @@ socket2 = { version = "0.4.7", optional = true, default-features = false }
 tokio = { version = "1.25.0", default-features = false }
 # Disable default-features to exclude unused dependency on libudev
 tokio-serial = { version = "5.4.4", optional = true, default-features = false }
-tokio-util = { version = "0.7.4", features = ["codec"] }
+tokio-util = { version = "0.7.4", default-features = false, features = [
+    "codec",
+] }
 
 [dev-dependencies]
 env_logger = "0.10.0"
 futures = "0.3.25"
-tokio = { version = "1.25.0", features = ["net", "macros", "io-util", "rt", "time"] }
+tokio = { version = "1.25.0", default-features = false, features = ["macros", "rt-multi-thread", "time"] }
+tokio-serial = { version = "5.4.4", default-features = false }
 
 [features]
-default = ["tcp", "rtu"]
-rtu = ["tokio-serial", "futures-util/sink"]
+default = ["rtu", "tcp"]
+rtu = ["futures-util/sink"]
 tcp = ["tokio/net", "futures-util/sink"]
-sync = ["tokio/rt"]
-server = ["futures", "socket2/all", "tokio/macros", "tokio/rt", "tokio/rt-multi-thread"]
-tcp-server-unstable = ["tcp", "server"]
+rtu-sync = ["rtu", "tokio-serial"]
```

Comment:
I'd use the `dep:` syntax like  `dep:tokio-serial` to make it easier to read and distinguish between features and dependencies.

Reply:
Will try. In previous versions Cargo complained when using `dep:` without any ambiguities.

Reply:
Worked.